apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ChatView.kt

```diff
@@ -1345,6 +1348,24 @@ fun BoxWithConstraintsScope.FloatingButtons(
         firstItemIsVisible = firstVisibleIndex == 0
       }
   }
+  LaunchedEffect(listState) {
+    snapshotFlow { listState.layoutInfo.visibleItemsInfo }
+      .collect { visibleItemsInfo ->
+        if (visibleItemsInfo.find { it.index == 0 } != null) {
```

Comment:
if you need to do something like this, you can user .filter() on snapshotFlow before .collect()